addons/udes_stock/models/stock_move.py

```diff
@@ -10,25 +10,26 @@
 
 # Map move state to the refactor stage.
 STAGES = {
-    'confirmed': 'confirm',
-    'waiting': 'confirm',
-    'assigned': 'assign',
-    'partially_available': 'assign',
-    'done': 'validate'
+    "confirmed": "confirm",
+    "waiting": "confirm",
+    "assigned": "assign",
+    "partially_available": "assign",
+    "done": "validate",
 }
 
+
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    u_grouping_key = fields.Char('Key', compute='compute_grouping_key')
+    u_grouping_key = fields.Char("Key", compute="compute_grouping_key")
 
     def _unreserve_initial_demand(self, new_move):
         """ Override stock default function to keep the old move lines,
             so there is no need to create them again
         """
-        self.mapped('move_line_ids')\
-            .filtered(lambda x: x.qty_done == 0.0)\
-            .write({'move_id': new_move, 'product_uom_qty': 0})
+        self.with_context(bypass_reservation_update=False).mapped("move_line_ids").filtered(
+            lambda x: x.qty_done == 0.0
```

Comment:
I don't think this change will work as qty_done is not zero, so this change will not have any effect. We probably want to create a new move line for the remainder on the new move instead anyway